src/VirtoCommerce.CustomerExportImportModule.Data/VirtoCommerce.CustomerExportImportModule.Data.csproj

```diff
@@ -28,7 +28,7 @@
         </PackageReference>
         <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
         <PackageReference Include="VirtoCommerce.CustomerModule.Data" Version="3.28.0" />
-        <PackageReference Include="VirtoCommerce.Platform.Data" Version="3.53.0" />
+        <PackageReference Include="VirtoCommerce.Platform.Data" Version="3.65.0" />
```

Comment:
Set updated platform version in the module.manifest also.

Reply:
Fixed